scss/utilities/mixins/molecules/_footer-menus.scss

```diff
@@ -0,0 +1,37 @@
+@charset "UTF-8";
+
+//
+// Footer Menu Mixins
+//
+// Mixins for the menus that appear in the global footer.
```

Comment:
Code comments are for every single mixin and component we write. There is no catch-all for a file. Please comment on every mixin you write. Be descriptive in the name and description citing the purpose and usage of the mixin.